homeassistant/components/melnor/const.py

```diff
@@ -0,0 +1,8 @@
+"""Constants for the melnor integration."""
+
+
+DOMAIN = "melnor"
+DEFAULT_NAME = "Melnor Bluetooth"
```

Comment:
I think we can drop `Bluetooth` here since it doesn't look like Melnor has any non-bluetooth products

https://melnor.com/shop/

Reply:
They've got a wifi enabled version that's branded slightly differently, which I happen to maintain too. https://github.com/vanstinator/hass-raincloud

I was going to deprecate it entirely because it's old and discontinued, and then they went and did another manufacturing run of them last summer 🤦🏻

I figured it's easy enough to change that string in the future.